widgetssdk/src/main/res/values/strings.xml

```diff
@@ -6,14 +6,12 @@
     <string name="glia_chat_in_queue_message">An MSR will be with you shortly.</string>
     <string name="glia_chat_operator_has_joined">%1$s has joined the conversation</string>
     <string name="glia_chat_delivered">Delivered</string>
-    <string name="glia_chat_upgraded_to_audio_call">Upgraded to Audio call</string>
-    <string name="glia_chat_upgraded_to_video_call">Upgraded to Video call</string>
+    <string name="glia_chat_upgraded_to_audio_call">Upgraded to audio call</string>
+    <string name="glia_chat_upgraded_to_video_call">Upgraded to video call</string>
     <string name="glia_chat_single_choice_card_hint">Tap on the answer above</string>
     <string name="glia_chat_not_started_hint">Send a message to start chatting</string>
-    <string name="glia_chat_file_download_success_message">File is downloaded successfully.</string>
-    <string name="glia_chat_file_download_fail_message">"File download is failed."</string>
-    <string name="glia_chat_file_open_file_title">"Open File"</string>
-    <string name="glia_chat_file_download_failed_msg">Unable to download the file at the moment, please try again later.</string>
```

Comment:
do we don't need these two strings anymore?  `glia_chat_file_download_failed_msg` looks in use in `ChatView.java`.

Reply:
Those are duplicated - glia_chat_file_download_failed_msg is not used

Reply:
glia_chat_file_download_failed_msg is used and is not removed - just moved lower

Reply:
see line 14